Source/loadsave.cpp

```diff
@@ -962,6 +996,53 @@ void SaveGame()
 	dwLen = codec_get_encoded_len(tbuff - SaveBuff);
 	pfile_write_save_file(szName, SaveBuff, tbuff - SaveBuff, dwLen);
 	mem_free_dbg(SaveBuff);
+#ifdef PIXEL_LIGHT

```

Comment:
seems this could use restoring tbuff to previous value after custom saving/loading ends, it didn't corrupt saves in this case but when moved to a different place, it would